### ISSUE TITLE
Layouts for Backend Preview not usable since 1.1

### ISSUE THREAD

**Issue by cvh-sitegeist** [FriendsOfTYPO3/content-blocks#324]
Since version 1.1  there is the new feature to use a layout by:

```
<f:layout name="Preview"/>

<f:section name="Content">
    <f:asset.css identifier="my-backend-styles" href="{cb:assetPath()}/preview.css"/>
    <div>My content</div>
</f:section>
```

If I want to use a layout on each content block like:
`EXT:sitepackage/Resources/Private/Templates/Layouts/ContentBlocks/Preview/Content/Preview.html`
```
<div class="grid-view-wrapper">
    <div class="grid">
        <f:render section="Content" />
    </div>
</div>
```

the content is rendered three times (in Header, Content, and Footer). This makes the use of a layout unusable not like it is meant to be.

**Comment by nhovratov** [comment]
this is fixed in 1.1.3. Can you verify it?

**Comment by cvh-sitegeist** [comment]
Yes it is fixed in 1.1.3. Thank you
